errors.js

```diff
@@ -0,0 +1,22 @@
+exports.handleSqlErrors = (err, req, res, next) => {
+    if (err.code === "22P02") {
+        res.status(400).send({ msg: "bad request" });
+    } else {
+        next(err);
+    }
+};
+
+exports.handleCustomErrors = (err, req, res, next) => {
+    if (err.status) {
+        res.status(err.status).send({ msg: err.msg });
+    } else {
+        next(err);
+    }
+};
+
+exports.handleServerErrors = (err, req, res, next) => {
+    console.log(err);
```

Comment:
This is the one console log that's totally fine to leave in - because if an error has reached the 500 stage, it's something we've not thought about and should be notified about